database/factories/ModelFactory.php

```diff
@@ -76,6 +78,13 @@
     ];
 });
 
+$factory->define(Setting::class, function ($faker) {
```

Comment:
Seems a bit random to add this in an unrelated pull request...

@jbrooksuk just do this externally?